src/test/strtol.cc

```diff
@@ -257,6 +257,21 @@ TEST(StrictIECCast, Error) {
     (void)strict_iec_cast<int>("1T", &err);
```

Comment:
Huh? `1T` is allowed right? 

```
build$ setfattr -n ceph.quota.max_bytes -v 1T ./mnt/foo/
build$ getfattr -n ceph.quota.max_bytes ./mnt/foo/
# file: mnt/foo/
ceph.quota.max_bytes="1099511627776"
```

I know this is out of scope of this PR, no worries but this something to be noted @batrick @vshankar

Reply:
`1T` is beyond the range of `int` type, will throw error.

Reply:
Ah gotcha, the used cast type is `int` there.